src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -425,41 +425,17 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             return;
         }
 
-        auto newOpacity = std::clamp(_settings.TintOpacity() + adjustment,
+        auto newOpacity = std::clamp(_settings.Opacity() + adjustment,
                                      0.0,
                                      1.0);
-        if (_settings.UseAcrylic())
-        {
-            try
-            {
-                _settings.TintOpacity(newOpacity);
 
-                if (newOpacity >= 1.0)
-                {
-                    _settings.UseAcrylic(false);
-                }

```

Comment:
We're putting a lot of trust in the platform that it will "do the right thing" when acrylic is enabled and set to 1.0.